code/obj/artifacts/artifactprocs.dm

```diff
@@ -207,6 +265,12 @@
 			src.visible_message("<span class='alert'>[user.name] burns the artifact with [ZIP]!</span>")
 			return 0
 
+	if(istype(W,/obj/item/device/igniter))
+		var/obj/item/device/igniter/IGNITER = W
+		src.ArtifactStimulus("elec", 700)
+		src.ArtifactStimulus("heat", 385)
+		src.visible_message("<span class='alert'>[user.name] sparks against \the [src] with \the [IGNITER]!</span>")
```

Comment:
```suggestion
		var/obj/item/device/igniter/igniter = W
		src.ArtifactStimulus("elec", 700)
		src.ArtifactStimulus("heat", 385)
		src.visible_message("<span class='alert'>[user.name] sparks against \the [src] with \the [igniter]!</span>")
```

Reply:
I mainly did this for formatting consistency with the other vars used like this, but, in hindsight, I should've just changed the formatting of the other var names.